common/app/views/fragments/header.scala.html

```diff
@@ -2,15 +2,17 @@
 @import common.{Edition, LinkTo, SubscribeLink}
 @import conf.Configuration
 @import conf.switches.Switches._
+@import conf.switches.Switches.NewTabletHeader
 
 @* TODO: When implementing the desktop design, aim to consolidate the html *@
 @fragments.newHeader(page)
 
 @hasSlimHeader = @{page.metadata.hasSlimHeader}
+@HideOnTablet = @{NewTabletHeader.isSwitchedOn}
 
 @if(!mvt.ABNewDesktopHeader.isParticipating) {
     <header id="header"
-            class="l-header u-cf @if(hasSlimHeader) {l-header--is-slim l-header--no-navigation} js-header"
+            class="l-header u-cf @if(HideOnTablet) {l-header--switch-on-hide-on-tablet} @if(hasSlimHeader) {l-header--is-slim l-header--no-navigation} js-header"
```

Comment:
Maybe use the `RenderClasses` here instead? An example of how to use it is [here](https://github.com/guardian/frontend/blob/9b4e3c2f33be9c888a4816af60fe23935315f5bf/common/app/views/fragments/atoms/genericMedia.scala.html#L7)